examples/minimal/packages/client-react-external-wallet/package.json

```diff
@@ -14,12 +14,14 @@
     "@latticexyz/common": "link:../../../../packages/common",
     "@latticexyz/dev-tools": "link:../../../../packages/dev-tools",
     "@latticexyz/store-sync": "link:../../../../packages/store-sync",
+    "@tanstack/react-query": "5.22.2",
     "contracts": "workspace:*",
+    "p-retry": "^5.1.2",
```

Comment:
To adapt the transaction sending extension in this example, `p-retry`, which is utilized in the common library, has been installed here. If the direction of this implementation proves to be suitable, I plan to move the custom client implementation to the common library later. Then, I can remove `p-retry` from here.